server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/EcoreFacade.java

```diff
@@ -162,6 +163,12 @@ public void initializeNotationElement(NotationElement element, EObject semanticE
 		return diagram.getElements().stream().filter(element -> element.getSemanticElement() == null)
 				.collect(Collectors.toList());
 	}
+	
+	public List<NotationElement> findOldShapes(EClassifier eClassifier) {
```

Comment:
This method can be removed; we still need to implement a reliable way to uniquely identify model-server elements